### ISSUE TITLE
Trikes stop moving when blocked

### ISSUE THREAD

**Issue by alercah** [OpenRA/OpenRA#14672]
This is on Playtest 20180102.

To reproduce:

* Start Harkonnen Mission 02a
* Select all units
* Attempt to move them around the map

Observed:

* The units move towards the destination, but when something (like another unit) blocks a trike, rather than find another path, it simply stops.

Expected:

* If the path is blocked by another unit, especially one in the same group, the unit should continue to the destination.

I have no reason to believe this is limited to trikes on this mission map, but this is where I encountered it.

**Comment by FrameLimiter** [comment]
Have you found this issue to be present on current release as well or just the playtest?
Perhaps a duplicate of #5968?

**Comment by alercah** [comment]
It is present in the release. So perhaps it is just a duplicate.

**Comment by ABrandau** [comment]
This bug has been on the engine for years, and it affects all the units with the mobile trait, its one of the reasons why the AI gets its harvesters stuck when placing refineries poorly.
